lib/cc/engine/analyzers/violation.rb

```diff
@@ -8,8 +8,8 @@ module Analyzers
       class Violation
         attr_reader :issue
 
-        def initialize(base_points, issue, hashes)
-          @base_points = base_points
+        def initialize(language, issue, hashes)
+          @language = language
```

Comment:
I don't really understand the reasoning behind `language_strategy` instead of `language`?